src/status_im/data_store/realm/schemas/account/chat.cljs

```diff
@@ -251,3 +251,9 @@
   (update v13 :properties assoc
           :group-chat-local-version {:type :int
                                      :optional true}))
+(def v15
+  (-> v14
+      (update :properties dissoc :message-overhead)
```

Comment:
`dissoc` is variadic, so you can have just

`(update v14 :properties dissoc :message-overhead :removed-from-at .....)`

Reply:
oh, thanks! will do

Reply:
done!